src/SQLQueryStress/Form1.cs

```diff
@@ -238,8 +238,8 @@ private void backgroundWorker1_RunWorkerCompleted(object sender, RunWorkerComple
 
         private void AutoSaveResults(string resultsAutoSaveFileName)
         {
-            string extension = Path.GetExtension(resultsAutoSaveFileName).ToLower();
-            if (extension == ".csv")
+            string extension = Path.GetExtension(resultsAutoSaveFileName).ToUpperInvariant();

```

Comment:
Is `.ToUpperInvariant()` needed, when the comparison now is case insensitive? Also it previously was `.ToLower()`

Reply:
This is the FxCop recommended approach